api/reqs/tests/views_policies_tests.py

```diff
@@ -180,3 +180,15 @@ def test_pk_id():
     mommy.make(Policy, pk=pk_id)
     response = client.get(path + '.json').json()
     assert response['id'] == pk_id
+
+
+@pytest.mark.django_db
+def test_slug():
+    client = APIClient()
+    slug = "hello-there"
+    path = "/policies/{0}".format(slug)
```

Comment:
Mind using f-strings, pretty please?
```python
path = f"/policies/{slug}/"
```

Reply:
Oh sure. Should I change the test above it to use f-strings too, since this new test is just a copy-paste of that one?

Reply:
No obligation, but that would be appreciated.